apps/web/app/partners.dub.co/(apply)/[programSlug]/apply/form.tsx

```diff
@@ -70,7 +70,7 @@ export function ProgramApplicationForm({
         });
 
         router.push(
-          `/apply/${program.slug}/application/success?${searchParams.toString()}`,
+          `/${program.slug}/apply/success?${searchParams.toString()}`,
```

Comment:
Not sure why this route was `/apply/[slug]/application/success` before instead of `/[slug]/apply/success`, or why that worked